tests/visualization_tests/matplotlib_tests/test_contour.py

```diff
@@ -235,3 +238,53 @@ def test_generate_contour_plot_for_few_observations(params: List[str]) -> None:
     study = prepare_study_with_trials(less_than_two=True)
     figure = plot_contour(study, params)
     assert not figure.has_data()
+
+
+def all_equal(iterable: Iterable) -> bool:
+    """Returns True if all the elements are equal to each other"""
+    return len(set(iterable)) <= 1
+
+
+def range_covers(range1: Tuple[float, float], range2: Tuple[float, float]) -> bool:
+    """Returns True if `range1` covers `range2`"""
+    min1, max1 = sorted(range1)
+    min2, max2 = sorted(range2)
+    return min1 <= min2 and max1 >= max2
+
+
+def test_contour_subplots_have_correct_axis_labels_and_ranges() -> None:
+    study_without_trials = prepare_study_with_trials()
```

Comment:
[question] Why did you name `study_without_trials` for the returned value of `prepare_study_with_trials` here? :eyes:

Reply:
Ah I copied `study_without_trials = prepare_study_with_trials(no_trials=True)`, and then removed `no_trials=True`. I'll fix.